packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -255,6 +262,15 @@ export interface ISummaryRuntimeOptions {
 export interface IContainerRuntimeOptions {
     summaryOptions?: ISummaryRuntimeOptions;
     gcOptions?: IGCRuntimeOptions;
+    /**
+     * Affects the behavior while loading the runtime when the data verification check which
+     * compares the DeltaManager sequence number (obtained from protocol in summary) to the
+     * runtime sequence number (obtained from runtime metadata in summary) finds a mismatch.
+     * 1. "close" (default) will close the container with an assertion.
+     * 2. "log" will log an error event to telemetry, but still continue to load.
+     * 3. "bypass" will skip the check entirely. This is not recommended.
+     */
+    loadSequenceNumberVerification?: "close" | "log" | "bypass";
```

Comment:
I am wondering if we need this. If the sequenceNumbers do not match, its a critical bug right. Even if we proceed and load the container, things will definitely break or we would have missing data.

Reply:
Yeah I was just thinking of troubleshooting/recovery scenarios, or future change in assumptions. I don't think we need to really advertise this option.